sdks/python/apache_beam/io/external/xlang_kinesisio_it_test.py

```diff
@@ -64,7 +64,7 @@
   DockerContainer = None
 # pylint: enable=wrong-import-order, wrong-import-position, ungrouped-imports
 
-LOCALSTACK_VERSION = '0.11.3'
+LOCALSTACK_VERSION = '3.8.1'
```

Comment:
New aws ios didn't seem to work with old localstack, so I had to update this and some of the other things in this file.